.github/workflows/ci_linting.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: source ./codebuild/bin/s2n_setup_env.sh
 
       - name: Check
-        run: ./codebuild/bin/copyright_mistake_scanner.sh
+        run: ./codebuild/bin/header_mistake_scanner.sh
```

Comment:
The github action job is still called "copyright". Should it maybe just be called "headers" now?